Agents/BuildingMatchingAgent/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+
+services:
+  java_agent_eg:
+    image: building_matching_agent:1.0.0
+    build: .
+    container_name: building_matching_agent
+    ports:
+      - "58085:8080"
+
+    volumes:
+      - external_maven_repo:/root/.m2
+
+
+volumes:
+  external_maven_repo:
+    name: "external_maven_repo"
```

Comment:
Is this volume required? I'm guessing it's just a convenience...

Reply:
Addressed by commit [ee3cc25](https://github.com/cambridge-cares/TheWorldAvatar/pull/616/commits/ee3cc255c196a2ed5e1ca514823e3f49596895d9)